app/frontend/src/search/data/locations.js

```diff
@@ -0,0 +1,145 @@
+export const locations = [
+    'Barking and Dagenham',
+    'Barnet',
+    'Bexley',
+    'Brent',
+    'Bromley',
+    'Camden',
+    'City of London',
+    'Croydon',
+    'Ealing',
+    'Enfield',
+    'Greenwich',
+    'Hackney',
+    'Hammersmith and Fulham',
+    'Haringey',
+    'Harrow',
+    'Havering',
+    'Hillingdon',
+    'Hounslow',
+    'Islington',
+    'Kensington and Chelsea',
+    'Kingston upon Thames',
+    'Lambeth',
+    'Lewisham',
+    'Merton',
+    'Newham',
+    'Redbridge',
+    'Richmond upon Thames',
+    'Southwark',
+    'Sutton',
+    'Tower Hamlets',
+    'Waltham Forest',
+    'Wandsworth',
+    'Westminster',
+    'Bath',
+    'Birmingham',
+    'Bradford',
+    'Brighton and Hove',
+    'Bristol',
+    'Cambridge',
+    'Canterbury',
+    'Carlisle',
+    'Chelmsford',
+    'Chester',
+    'Chichester',
+    'Coventry',
+    'Derby',
+    'Exeter',
+    'Gloucester',
+    'Hereford',
+    'Kingston-upon-Hull',
+    'Lancaster',
+    'Leeds',
+    'Leicester',
+    'Lichfield',
+    'Lincoln',
+    'Liverpool',
+    'London',
+    'Manchester',
+    'Newcastle',
+    'Norwich',
+    'Nottingham',
+    'Oxford',
+    'Peterborough',
+    'Plymouth',
+    'Portsmouth',
+    'Preston',
+    'Ripon',
+    'Salford',
+    'Salisbury',
+    'Sheffield',
+    'Southampton',
+    'St Albans',
+    'Stoke-on-Trent',
+    'Sunderland',
+    'Truro',
+    'Wakefield',
+    'Wells',
+    'Winchester',
+    'Wolverhampton',
+    'Worcester',
+    'York',
+    'Avon',
+    'Bedfordshire',
+    'Berkshire',
+    'Bristol',
+    'Buckinghamshire',
+    'Cambridgeshire',
+    'Cheshire',
+    'Cleveland',
+    'Cornwall',
+    'County Durham',
+    'Cumbria',
+    'Derbyshire',
+    'Devon',
+    'Dorset',
+    'East Sussex',
+    'East Yorkshire',
+    'Essex',
+    'Gloucestershire',
+    'Greater Manchester',
+    'Hampshire',
+    'Herefordshire',
+    'Hertfordshire',
+    'Isle of Wight',
+    'Isles of Scilly',
+    'Kent',
+    'Lancashire',
+    'Leicestershire',
+    'Lincolnshire',
+    'Merseyside',
+    'Middlesex',
+    'Norfolk',
+    'North East Lincolnshire',
+    'North Lincolnshire',
+    'North Yorkshire',
+    'Northamptonshire',
+    'Northumberland',
+    'Nottinghamshire',
+    'Oxfordshire',
+    'Rutland',
+    'Shropshire',
+    'Somerset',
+    'South Gloucestershire',
+    'South Yorkshire',
+    'Staffordshire',
+    'Suffolk',
+    'Surrey',
+    'Teesside',
+    'Tyne and Wear',
+    'Warwickshire',
+    'West Midlands',
+    'West Sussex',
+    'West Yorkshire',
+    'Wiltshire',
+    'Worcestershire',
+    'London',
+    'South East',
+    'Yorkshire and the Humber',
+    'East of England',
+    'West Midlands',
+    'North West',
+    'North East',
+    'South West',
+    'East Midlands'];
```

Comment:
this whole file is duplicated from config yaml files that the serverside code use. cant really avaoid this duplication for now unless we automate something in CI. but for now i dont see this as a huge overhead

Reply:
Further to my alphabetising comment, above - could we just expose these from the backend in a API?  I'd prefer we didn't have to maintain two sources of truth...

Reply:
https://dfedigital.atlassian.net/browse/TEVA-794 created to address this